src/task/task.mapper.ts

```diff
@@ -14,6 +14,11 @@ export class TaskMapper {
 		};
 	}
 
+	public toDTOs(entities: Task[]): TaskDTO[] {
+		const task: TaskDTO[] = entities;
+		return task;
```

Comment:
In this case, .toDTOs() is not resolving our problem. We added Mapper in order to avoid cast operation. If TaskEntity has other properties than TaskDTO, the cast will not be possible.
See a possible solution:

let tasks: TaskDTO[] = [];
for (const entity of entities) {
  tasks.push(this.toDTO(entity))
}
return tasks;